README.md

```diff
@@ -179,3 +179,68 @@ spec:
  nodeSelector:
    master: eno3
 ```
+
+### Customized Injections
+
+Customized injections allows user to define additinal injections (besides what's supported in NRI, such as ResourceName, Downward API volumes etc) in Kubernetes ConfigMap and request additional injection for individual pod based on pod label. Currently customized injection only support injecting pod annotations.
+
+In order to use this feature, user needs to create the customized injection ConfigMap with name `nri-customized-injections` in namespace `kube-system`. The data entry in ConfigMap is in the format of key:value pair. Key is a user defined label that will be used to match with pod labels, Value is the actual injection that will be applied to pod manifest. NRI would listen to the creation/update/deletion of this ConfigMap and update its internal data structure every 30 seconds so that subsquential creation of pods will be evaluated against the latest customized injections.
+
+Below is an example of customized injection ConfigMap:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-customized-injections
+  namespace: kube-system
+data:
+  feature.pod.kubernetes.io/sriov-network: '{"op": "add", "path": "/metadata/annotations", "value": {"k8s.v1.cni.cncf.io/networks": "sriov-net-attach-def"}}'
+```
+
+`feature.pod.kubernetes.io/sriov-network` is a user defined label to request additional networks. Every pod that contains this label will be applied with the patch that's defined in the following json string.
```

Comment:
nit: contain this  label with a value set to `true`

Reply:
Done.